e2e/devtools.test.js

```diff
@@ -0,0 +1,385 @@
+import DevTools from '../packages/devtools/src/index'
```

Comment:
what do you think on splitting this files into several ones to run tests in parallel?

Reply:
I wasn't sure how Jest is doing parallelization but it seems to work 👍 split test suite into 2 files